rm/src/main/java/io/seata/rm/AbstractRMHandler.java

```diff
@@ -153,7 +152,7 @@ public AbstractResultMessage onRequest(AbstractMessage request, RpcContext conte
 
     @Override
     public void onResponse(AbstractResultMessage response, RpcContext context) {
-        throw new ShouldNeverHappenException();
+
```

Comment:
Why remove here?

Reply:
It may have been removed by hand, I will roll back.